ref/general/microshed.adoc

```diff
@@ -1,23 +1,23 @@
 :page-layout: general-reference
 :page-type: general
-:page-description: MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+:page-description: MicroShed testing helps you to write integration tests using Testcontainers framework for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 :page-categories: MicroShed testing
 :seo-title: Testing in a container with MicroShed testing
-:seo-description:  MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+:seo-description:  MicroShed testing helps you to write integration tests using Testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 = Testing microservice applications
 
-Microshed testing is a Java library that helps you write true-to-production integration tests for your microservice applications using link:https://openliberty.io/blog/2019/03/27/integration-testing-with-testcontainers.html[testcontainers].
-link:https://microshed.org/microshed-testing/[MicroShed Testing] tests your application from outside without accessing it internally.
+MicroShed Testing is a Java library that helps you write true-to-production integration tests for your microservice applications with link:https://openliberty.io/blog/2019/03/27/integration-testing-with-testcontainers.html[Testcontainers] framework.
+link:https://microshed.org/microshed-testing/[MicroShed Testing] framework analyzes your application from outside the link:https://www.docker.com/why-docker[Docker] container without accessing the application internally.
```

Comment:
Is microshed a Java library or a framework that uses the Testcontainers Java Library? I don't know, but asking because both are mentioned here

Reply:
also might need "the" before microshed

Reply:
also- are you sure about _without accessing internally_ ? just checking because although it tests _from_ the outside, still might access application code.

Reply:
> 
> 
> Is microshed a Java library or a framework that uses the Testcontainers Java Library? I don't know, but asking because both are mentioned here

MicoShed Testing is an extension of the Testcontainers Java library.Andy created MicroShed Testing Java library for Java EE and MicroProfile developers to test their applications.

Reply:
> _without accessing internally_

I changed it to 'without acessing the application internals' which is what the guide says.